app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -204,4 +221,19 @@ public function getCategoryById($categoryId)
     {
         return isset($this->categoriesCache[$categoryId]) ? $this->categoriesCache[$categoryId] : null;
     }
+
+    /**
+     * Standardize a string by lowercase-ing it
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function standardizeString($string)
+    {
+        if (function_exists('mb_strtolower')) {
```

Comment:
I believe `composer.json` for Magento already requires `ext-mbstring` so you may remove this check.